jscomp/main/bsb_main.ml

```diff
@@ -167,22 +169,16 @@ module CLI = struct
 
   let init_arg =
     let docv = "init path" in
-    let doc =
-      "Init sample project to get started. \n\
-       Note: (`bsb -init sample` will create a sample project while \n\
-       `bsb -init .` will reuse current directory)" in
+    let doc = "Currently does nothing. Enabled for compatibility." in
```

Comment:
could we also add the theme link in these docstrings?

Reply:
done!